app/components/decisions/DecisionTranslation.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div>
+        <div>
+            <table>
+                <thead>
+                    <tr>
+                        <td>English</td>
+                        <td>
+                            <select class="form-control" id="languages" v-model="selectedLanguage">
+                                <option 
+                                    v-for="lang in Object.keys(languages)"
+                                    :key="lang" 
+                                    :value="lang">
+                                    {{languages[lang]}}
+                                </option>
```

Comment:
can iterate object directly. non need of Object.keys...
btw internaly we always use locale for the language code   
```suggestion
                                <option 
                                    v-for="(language, locale) in languages"
                                    :key="locale" 
                                    :value="locale">
                                    {{language}}
                                </option>
```

Reply:
Done!